index.html

```diff
@@ -0,0 +1,193 @@
+          </button>
+          <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav gap-md-5 ms-auto mb-2 mb-lg-0 text-center">
+              <li class="nav-item">
+                <a
+                  class="nav-link"
+                  aria-current="page"
+                  href="./interfaces/about.html"
+                  >About</a
+                >
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#program">Program</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#speakers">Speakers</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#partners">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">News</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link primary-border mx-auto campaing" href="#"
+                  >DB Campaign</a
+                >
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <!-- THE HERO SECTION -->
+    <section class="hero-section">
+      <div class="container-fluid">
+        <div class="container-md">
+          <div class="hero-content">
+            <p class="red-text">"Welcome To DevBox!!"</p>
+            <h1 class="red-heading">
+              The Global Conference For Software Engineers 2022
+            </h1>
+            <div class="bg-secondry border border-white p-2 my-4">
+              <p>
+                Experience a new type of gathering, where all creative people
+                meet from over 80 countries is taking place in October, in
+                Canada.
+              </p>
+            </div>
+            <p class="attention"><strong>2022.10.21(THU) ~ 22(FRI)</strong></p>
+            <p>@ National Museum of Canada, Art Center Nabi and more.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- PROGRAM SECTION -->
+    <section id="program">
+      <div class="container-fluid">
+        <div class="container-md text-center">
+          <h2>Main Program</h2>
+          <hr class="separator" />
+          <ul class="cards"></ul>
+          <button
+            type="button"
+            class="btn btn-primary text-white d-block mx-auto"
+          >
+            Join The DB Global Conference 2022
+          </button>
+        </div>
+      </div>
+    </section>
+
+    <!-- SPEAKERS SECTION -->
+    <section id="speakers">
+      <div class="container-fluid">
+        <div class="container-md text-center">
+          <h2>Featured Speakers</h2>
+          <hr class="separator" />
+          <ul class="cards"></ul>
+        </div>
+      </div>
+    </section>
+
+    <!-- PARTNER SECTION -->
+    <section id="partners" class="bg-dark d-md-none">
```

Comment:
- [ ] Kindly add a partners section also for a desktop screen on the index page. Currently, it's only visible in the mobile version.

![image](https://user-images.githubusercontent.com/92215878/190702030-e6551e90-6e3e-4a90-9090-c756b1650fd4.png)

Reply:
I'll do so🐱‍🏍, thank you 👏.